src/framework/preferences/css/stylus/Enactors.styl

```diff
@@ -25,9 +25,11 @@
     font-family: "Comic Sans MS", sans-serif !important;
 }
 
-// UI Enhancer "Enlarge text" link options
+// UI Enhancer "Enlarge text" input/button options
 .fl-text-larger input,
-.fl-text-larger button {
+.fl-text-larger button,
+.fl-text-larger .fl-input-enhance,
+.fl-text-larger .fl-button-enhance, {
```

Comment:
Sorry about the confusion, @jobara. I'll try to clarify what I meant. 

I'll start with the other `fl-input-enhance` class that I have no question with. This css class is attached with on/off toggles so that when the preference "make input larger" is selected, the toggle labels will be enlarged. This class is used in html templates for preferences framework.

However with `fl-button-enhance`, it doesn't seem to be used in any templates or applied by any enactor. I did a quick search within "infusion-docs" repo and not seeing it to be suggested to integrators on their web pages as well. So I wonder if I miss something to understand why the use of this class is supported in css.

Let me know if my explanation is still not clear enough.